necsim/impls/no-std/src/cogs/dispersal_sampler/in_memory/packed_alias/dispersal.rs

```diff
@@ -37,8 +38,11 @@ impl<M: MathsCore, H: Habitat<M>, G: RngCore<M>> DispersalSampler<M, H, G>
         let alias_dispersals_at_location =
             unsafe { &self.alias_dispersal_buffer.get_unchecked(alias_range) };
 
-        let dispersal_target_index: usize =
-            AliasMethodSamplerAtom::sample_event(alias_dispersals_at_location, rng);
+        let dispersal_target_index: usize = AliasMethodSamplerAtom::sample_event(
+            alias_dispersals_at_location,
+            rng,
+            ClosedUnitF64::one(),
```

Comment:
Add a wrapper function to inline the one-factor more easily and hide that complexity when not needed